presto-cache/src/main/java/com/facebook/presto/cache/filemerge/FileMergeCacheConfig.java

```diff
@@ -44,19 +41,6 @@ public FileMergeCacheConfig setMaxCachedEntries(int maxCachedEntries)
         return this;
     }
 
-    public DataSize getMaxInMemoryCacheSize()
```

Comment:
This is different from what cache size mean for Alluxio. We should keep the separate. This defines how much memory you could use for the cache manager.

Reply:
Ah my bad! Thats why I was wondering why it is not configured in the configs.